inventory/packages/apt_deb.go

```diff
@@ -145,6 +145,10 @@ func parseAptUpdates(data []byte, showNew bool) []PkgInfo {
 			// This is a newly installed package and not an upgrade, ignore if showNew is false.
 			continue
 		}
+		// Drop trailing "[]" if they exist.
+		if bytes.Contains(pkg[len(pkg)-1], []byte("[]")) {
+			pkg = pkg[:len(pkg)-1]
+		}
```

Comment:
how did we discover this? e2etests?

Reply:
Yeah, came out of nowhere, never seen it before